src/test/kotlin/org/opensearch/commons/alerting/model/DocLevelMonitorInputTests.kt

```diff
@@ -42,14 +42,19 @@ class DocLevelMonitorInputTests {
     }
 
     @Test
-    fun `test create Doc Level Query with invalid characters for name`() {
-        val badString = "query with space"
+    fun `test create Doc Level Query with invalid name length`() {
+        val stringBuilder = StringBuilder()
+        repeat(256) {
+            stringBuilder.append("a")
+        }
+        val badString = stringBuilder.toString() + "b".repeat(256)
```

Comment:
Minor: Looks like this is testing a string of length 512. Would be better to test at the limit (257) imo

Reply:
changed testing to 257 chars